hw/formal/tools/dvsim/common_fpv_cfg.hjson

```diff
@@ -2,22 +2,22 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  flow:             fpv
+  flow:             formal
```

Comment:
Now it looks except `sub_flow` & the `design_level` override, both files are identical? See if you can consolidate further meaningfully.